Std/Data/Nat/Lemmas.lean

```diff
@@ -230,6 +230,9 @@ theorem sub_eq_sub_min (n m : Nat) : n - m = n - min n m := by
 protected theorem mul_right_comm (n m k : Nat) : n * m * k = n * k * m := by
   rw [Nat.mul_assoc, Nat.mul_comm m, ← Nat.mul_assoc]
 
+protected theorem mul_cross_comm (a b c d : Nat) : (a * b) * (c * d) = (a * c) * (b * d) := by
```

Comment:
Is this what the theorem is called in mathlib? I would have guessed it is something more like `mul_mul_mul_comm`.

Reply:
You're right, it is called `mul_mul_mul_comm`.